packages/jest-cli/src/lib/handleDeprecationWarnings.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+'use strict';
+
+const chalk = require('chalk');
+const {KEYS} = require('../constants');
+
+module.exports = (
```

Comment:
do you mind adding a line or two explaining why we need this? 🙂
the callsite doesn't really give any information either